src/XR/features/WebXRMeshDetector.ts

```diff
@@ -238,13 +231,23 @@ export class WebXRMeshDetector extends WebXRAbstractFeature {
                         mesh.normals[i + 2] = -1 * xrMesh.normals[i + 2];
                     }
                 }
-            }
-            else {
+            } else {
                 mesh.positions = xrMesh.positions;
-                mesh.indices = xrMesh.indices;
                 mesh.normals = xrMesh.normals;
             }
 
+            if (xrMesh.isClockwiseWindingOrder) {
+                // Babylon.js expects counter clockwise winding order for meshes

```

Comment:
Babylon.js winding order is configurable in the material as `sideOrientation` or can be overridden with the `overrideMaterialSideOrientation` property.

Reply:
Is that solving the issue here? Using the sideOrientation of the material / mesh.overrideMaterialSideOrientation ?

Reply:
that could solve the problem. If setting a material property is a discoverable/known way to fix winding orders we could do that instead. I think we will still need to expose this value in the WebXRVertexData type so that folks know to use/not use this flag.

Reply:
granted, after talking with OpenXR owners they also felt we could standardize on counter clockwise winding order for indices provided from webxr. @bghgary  and @RaananW  what would you two prefer?

Reply:
I'm not sure I'm the right person for an opinion for this. I don't have all the context. Is this part of the proposed contract for WebXR or Babylon.js? Maybe we can chat offline a bit.

Reply:
I think we should do the following:

1. Assume that WebXR provides mesh coordinates in a right handed coordinate system with counterclockwise winding order.
2. Update the WebXR mesh detector to not assume a clockwise winding order for right handed coordinate systems like it does currently.

The pr is now up to date with changes based on these two principles